pkg/gardenlet/controller/seed/seed_control.go

```diff
@@ -265,26 +257,21 @@ func (r *reconciler) reconcile(ctx context.Context, gardenClient client.Client,
 
 		parentLogMessage := "Can't delete Seed, because the following objects are still referencing it:"
 		if len(associatedShoots) != 0 {
-			message := fmt.Sprintf("%s Shoots=%v", parentLogMessage, associatedShoots)
-			log.Info(message)
-			r.recorder.Event(seed, corev1.EventTypeNormal, v1beta1constants.EventResourceReferenced, message)
+			log.Info("Cannot delete SEed because the following Shoots are still referencing it", "shoots", associatedShoots)
+			r.recorder.Event(seed, corev1.EventTypeNormal, v1beta1constants.EventResourceReferenced, fmt.Sprintf("%s Shoots=%v", parentLogMessage, associatedShoots))
 		}
 		if len(associatedBackupBuckets) != 0 {
-			message := fmt.Sprintf("%s BackupBuckets=%v", parentLogMessage, associatedBackupBuckets)
-			log.Info(message)
-			r.recorder.Event(seed, corev1.EventTypeNormal, v1beta1constants.EventResourceReferenced, message)
+			log.Info("Cannot delete SEed because the following BackupBuckets are still referencing it", "backupBuckets", associatedBackupBuckets)
```

Comment:
```suggestion
			log.Info("Cannot delete Seed because the following BackupBuckets are still referencing it", "backupBuckets", associatedBackupBuckets)
```